rollup.config.mjs

```diff
@@ -0,0 +1,20 @@
+import pkg from './package.json';
+
+export default {
+  input: 'index.mjs',
+  external: (
+    Object.keys (pkg.peerDependencies)
+    .concat (Object.keys (pkg.dependencies))
+  ),
+  output: {
+    format: 'umd',
+    name: 'flutureFastify',
+    file: 'index.js',
+    interop: false,
+    globals: {
+      'fluture': 'fluture',
```

Comment:
This is wrong, but also not needed. You don't need to make a umd out of this, because it's not intended for browsers. Just do cjs, and you can drop the name, the globals, and probably other stuff.